worlds/sc2wol/LogicMixin.py

```diff
@@ -17,17 +17,21 @@ def _sc2wol_has_air_anti_air(self, multiworld: MultiWorld, player: int) -> bool:
                or get_option_value(multiworld, player, 'required_tactics') > 0 and self.has('Wraith', player)
 
     def _sc2wol_has_competent_anti_air(self, multiworld: MultiWorld, player: int) -> bool:
-        return self.has_any({'Marine', 'Goliath'}, player) or self._sc2wol_has_air_anti_air(multiworld, player)
+        return self.has('Goliath', player) \
+                or self.has('Marine', player) and self.has_any({'Medic', 'Medivac'}, player) \
+                or self._sc2wol_has_air_anti_air(multiworld, player)
 
     def _sc2wol_has_anti_air(self, multiworld: MultiWorld, player: int) -> bool:
-        return self.has_any({'Missile Turret', 'Thor', 'War Pigs', 'Spartan Company', "Hel's Angel", 'Battlecruiser', 'Wraith'}, player) \
+        return self.has_any({'Missile Turret', 'Thor', 'War Pigs', 'Spartan Company', "Hel's Angel", 'Battlecruiser', 'Marine', 'Wraith'}, player) \
                 or self._sc2wol_has_competent_anti_air(multiworld, player) \
                 or get_option_value(multiworld, player, 'required_tactics') > 0 and self.has_any({'Ghost', 'Spectre'}, player)
 
     def _sc2wol_defense_rating(self, multiworld: MultiWorld, player: int, zerg_enemy: bool, air_enemy: bool = True) -> bool:
         defense_score = sum((defense_ratings[item] for item in defense_ratings if self.has(item, player)))
         if self.has_any({'Marine', 'Marauder'}, player) and self.has('Bunker', player):
             defense_score += 3
+        if self.has_all({'Siege Tank', 'Maelstrom Rounds'}, player):
```

Comment:
well, I guess nevermind, I found where it applies.

Reply:
Added a comment to the defense rating table explaining that the commented defense ratings are handled in LogicMixin, as you're the second person to ask about that.  I'm counting that as a bugfix, as it was possible for Maelstrom Rounds to apply to a mission that did not have Siege Tanks predeployed.